XCUITests/FxScreenGraph.swift

```diff
@@ -53,17 +53,28 @@ let allIntroPages = [
 ]
 
 let HomePanelsScreen = "HomePanels"
+let PrivateHomePanelsScreen = "PrivateHomePanels"
 let HomePanel_TopSites = "HomePanel.TopSites.0"
 let HomePanel_Bookmarks = "HomePanel.Bookmarks.1"
 let HomePanel_History = "HomePanel.History.2"
 let HomePanel_ReadingList = "HomePanel.ReadingList.3"
+let P_HomePanel_TopSites = "P_HomePanel.TopSites.0"
+let P_HomePanel_Bookmarks = "P_HomePanel.Bookmarks.1"
+let P_HomePanel_History = "P_HomePanel.History.2"
+let P_HomePanel_ReadingList = "P_HomePanel.ReadingList.3"
```

Comment:
I'm not sure I like this. 

Looking at the graph it makes it extremely messy.

With the `ScreenGraph` in this current state, I'm not sure what the alternative is, but I think we should be prepared for some further work on this.

Reply:
Oh trust me, I don't like this either - only if each node has a state where it can be marked private/non-private ... wait, I think that's your proposal is about 👍

Reply:
:+1: